ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -288,6 +290,20 @@ public ExecutionPayloadHeader deserializeJsonExecutionPayloadHeader(
         .jsonDeserialize(objectMapper.createParser(jsonFile));
   }
 
+  public SszList<SignedValidatorRegistration> deserializeSignedValidatorRegistrations(
```

Comment:
The other deserialise methods here are detecting based on slot automatically so I'm not sure I'd add this here since it's not actually tied to a milestone - it's more tied to the API version.

I actually wonder if this is a case where the schema shouldn't be part of SchemaDefinitions. If we put it in SchemaDefinitions we can only change the API at milestone boundaries but I'm pretty sure that's not how the API is going to evolve. And if it wasn't in SchemaDefinitions we could make this API work regardless of whether Bellatrix is enabled (though it probably wouldn't have any actual effect until Bellatrix was active).

Reply:
yep, that was my feeling. Going to move them in API itself